Fedora/README.md

```diff
@@ -0,0 +1,13 @@
+##Its Strengthes :)
+--- 
+* Easy to use, simple, and beginner friendly 
+* 100% free 
+* Less hardware-consuming as it consumes ONLY 1GB of RAM
+* Customizable
+* Flatpak is installed by default on Fedora 
+
+##Its weaknesses :(
+---
+* Requires long time to install
+* Execluded non-FOSS software and doesn't include softwares with usage restrictions 
```

Comment:
This could be phrased in a clearer way
For example "Does not include proprietary software and software with usage restrictions"